go.mod

```diff
@@ -12,8 +12,40 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-go 1.18
+go 1.19
```

Comment:
Project Go version is still 1.19, should be 1.20

Reply:
The following error was produced when I tried using 1.20:

`go.mod file indicates go 1.20, but maximum supported version is 1.19`

Reply:
You may not have Go 1.20 installed - run `go version` to see what version you're using locally.